test/trello-provider.test.ts

```diff
@@ -0,0 +1,146 @@
+/* Copyright © 2021 Seneca Project Contributors, MIT License. */
+
+import * as Fs from 'fs'
+
+import TrelloProvider from '../src/trello-provider'
+
+
+const Seneca = require('seneca')
+const SenecaMsgTest = require('seneca-msg-test')
+const TrelloProviderMessages = require('./trello-provider.messages').default
+
+const CONFIG: any = {}
+
+if (Fs.existsSync(__dirname + '/local-config.js')) {
+    Object.assign(CONFIG, require(__dirname + '/local-config.js'))
+}
+
+jest.setTimeout(10000)
+
+describe('trello-provider', () => {
+
+    test('happy', async () => {
+        const seneca = Seneca({legacy: false})
+            .test()
+            .use('promisify')
+            .use('provider', {
+                provider: {
+                    trello: {
+                        keys: {
+                            api: {
+                                value: [CONFIG.key, CONFIG.token],
```

Comment:
let's use a sub object with named fields here, instead of an array, as it is better for "self" documenting

Reply:
I see, done! Thank you for the suggestion.